pages/docs/configuration/configuration-mapping.md

```diff
@@ -50,61 +56,104 @@ Furthermore, mappings come int two types of `constraint`: `consistent` and `cons
 
-Furthermore, mappings have an optional parameter `timing`, it can be:
+For a sequential participant, any combination of `read`/`write`-`consistent/conservative` is valid. For a participant running in parallel, only `read`-`consistent` and `write`-`conservative` is possible. More details are given [further below](configuration-mapping.html#restrictions-for-parallel-participants).
 
-* `initial` (the default): The mapping is only computed once, the first time it is used. This is sufficient for stationary meshes (also including the reference mesh in an Lagrangian or an ALE description).
-* `onadvance`: The mapping is newly computed for every mapping of coupling data. This can be expensive and is only recommend if you know exactly why you want to do this.
+## Available mapping methods in preCICE
 
-Concerning mapping methods, preCICE offers four variants:
+The mapping method itself is defined in the xml configuration after the colon `mapping:...` (e.g. `<mapping:nearest-neighbor ...`). In general, preCICE offers two broader groups of mapping methods
 
-* `nearest-neighbor`: A first-order method, which is fast, easy to use, but, of course, has numerical deficiencies.
-* `nearest-projection`: A (mostly) second-order method, which first projects onto mesh elements and, then, uses linear interpolation within each element (compare the figure below). The method is still relatively fast and numerically clearly superior to `nearest-neighbor`. The downside is that mesh connectivity information needs to be defined, i.e. in the adapter, the participant needs to tell preCICE about edges in 2D and edges, triangles, or quads (see [issue](https://github.com/precice/precice/issues/153)) in 3D. On the [mesh connectivity page](couple-your-code-defining-mesh-connectivity.html), we explain how to do that. If no connectivity information is provided, `nearest-projection` falls back to an (expensive) `nearest-neighbor` mapping.
-* `linear-cell-interpolation`: A second-order method similar to `nearest-projection`, but designed for volumetric coupling, where the coupling mesh is a region of space and not a domain boundary. It interpolates on triangles in 2D and on tetrahedra in 3D. If none are found, it falls back on `nearest-projection`.
-* `nearest-neighbor-gradient`: A second-order method, which uses the same algorithm as nearest-neighbor with an additional linear approximation using gradient data. This method requires additional gradient data information. On the [gradient data page](couple-your-code-gradient-data.html), we explain how to add gradient data to the mesh. This method is only applicable with the `consistent` constraint.
-* Radial-basis function mapping. Here, the configuration is more involved, so keep reading.
+![Mapping options](images/docs/configuration/doc-mapping-options.svg)
+
+Methods marked in orange have been added more recently in preCICE and are available as annotated by the corresponding preCICE version.
```

Comment:
Since you are giving the versions, I don't think that the colors are important. What is "recent"? When should we un-color these bubbles?

Reply:
Let's make it more timeless